### ISSUE TITLE
Cleanup doesn't clean up the export directory

### ISSUE THREAD

**Issue by iainlane** [ximion/appstream-generator#41]
I just dropped powerpc from the zesty config, because it's deleted in the archive. The output files are still there, just not updated any more.

I wonder if asgen could take care of cleaning up the output directory? That is - delete everything in there that isn't either currently being updated or immutable.

Good idea or bad?

**Comment by ximion** [comment]
Good idea, I think - originally I intentionally didn't clean up that stuff because it might still be useful, but since we automatically clean up data in the `media/` directory as well I think *not* removing the exported YAML/XML is the actual unexpected result.
